### PR TITLE
signature check: report to Slack

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
If a tree falls in the forest and all that.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ci/bash-lib.yml [hunk 1]
```diff
@@ -100,3 +100,4 @@ steps:
     }
     END
     echo "##vso[task.setvariable variable=${{parameters.var_name}}]$TMP"
+  displayName: install Bash lib
```
Comment (garyverhaegen-da) [inline ci/bash-lib.yml:103]: Not technically related, but in looking at the job result for the past few days I noticed this job is reported as "Bash" in the Azure UI and that makes it harder to understand.

**Review by cocreature — APPROVED** [review]
Thanks!

**Review by SamirTalwar-DA — APPROVED** [review]
Sounds sensible to me.

**Comment by SamirTalwar-DA** [comment]
Has this failed recently, or is this more precautionary?

**Comment by garyverhaegen-da** [comment]
> Has this failed recently, or is this more precautionary?

As far as I'm aware it's been successful on every run since it was introduced, but that was only last Thursday. Honestly if this (legitimately) fails we're in serious trouble.

**Comment by SamirTalwar-DA** [comment]
I am sold.

**Pull request merged**